src/pages/multipleProductValidity.tsx

```diff
@@ -40,113 +39,148 @@ const MultipleProductValidity = ({
     multipleProducts,
     errors,
     csrfToken,
-}: MultipleProductValidityProps): ReactElement => (
-    <FullColumnLayout title={title} description={description} errors={errors}>
-        <CsrfForm
-            action="/api/multipleProductValidity"
-            method="post"
-            className="multiple-product-validity-page"
-            csrfToken={csrfToken}
-        >
-            <>
-                <ErrorSummary errors={errors} />
-                <div className={`govuk-form-group ${errors.length > 0 ? 'govuk-form-group--error' : ''}`}>
-                    <h1 className="govuk-heading-l" id="multiple-product-validity-page-heading">
-                        When does the product expire?
-                    </h1>
-                    <span className="govuk-hint" id="operator-products-hint">
-                        {operatorName} - {numberOfProducts} products - {upperFirst(passengerType)}
-                    </span>
-                    <span className="govuk-hint" id="multiple-product-validity-page-hint">
-                        We need to know the time that this product would be valid until
-                    </span>
-                    <span className="govuk-hint" id="24hr-validity-type-hint">
-                        24hr means a ticket purchased at 3pm will be valid until 3pm on its day of expiry
-                    </span>
-                    <span className="govuk-hint" id="calendar-validity-type-hint">
-                        Calendar day means a ticket purchased at 3pm would be valid until midnight on its day of expiry
-                    </span>
-                    <FormElementWrapper errors={errors} errorId={errorId} errorClass="govuk-radios--error">
+    endTimesList,
```

Comment:
If we default this to an empty array here, then we don't have to do || [] on using it.